examples/Tutorial.ipynb

```diff
@@ -0,0 +1,1262 @@
+      "0         0.679960        0.009428                5            0.661538   \n",
+      "1         0.919138        0.051844                3            0.938462   \n",
+      "2         0.939394        0.085710                2            0.969231   \n",
+      "3         0.939971        0.065734                1            0.938462   \n",
+      "4         0.820067        0.023589                4            0.846154   \n",
+      "\n",
+      "   split1_train_score  split2_train_score  mean_train_score  std_train_score  \n",
+      "0            0.701493            0.676471          0.679834         0.016484  \n",
+      "1            0.985075            0.941176          0.954904         0.021362  \n",
+      "2            0.985075            0.970588          0.974965         0.007170  \n",
+      "3            0.970149            0.970588          0.959733         0.015042  \n",
+      "4            0.910448            0.867647          0.874750         0.026724  \n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('split0_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True\n",
+      "  warnings.warn(*warn_args, **warn_kwargs)\n",
+      "/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('split1_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True\n",
+      "  warnings.warn(*warn_args, **warn_kwargs)\n",
+      "/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('split2_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True\n",
+      "  warnings.warn(*warn_args, **warn_kwargs)\n",
+      "/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('mean_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True\n",
+      "  warnings.warn(*warn_args, **warn_kwargs)\n",
+      "/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('std_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True\n",
+      "  warnings.warn(*warn_args, **warn_kwargs)\n"
+     ]
+    }
+   ],
+   "source": [
+    "grid_search_df = pd.DataFrame(grid_search_results)\n",
+    "print(grid_search_df)"
```

Comment:
In the notebook, I see the following `FutureWarning` here:

```
FutureWarning: You are accessing a training score ('split2_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True
```

Is there a way to address this? Or maybe just suppress the warning?